charts/keycloak/values.yaml

```diff
@@ -363,7 +370,7 @@ pgchecker:
 
 postgresql:
   # If `true`, the Postgresql dependency is enabled
-  enabled: true
+  enabled: false
```

Comment:
I'd keep postgresql as the default and disable the sqlserver dependency by default.